docusaurus/docs/reference/30-configuration/controller.md

```diff
@@ -510,10 +510,12 @@ The raft section enables running multiple controllers in a cluster.
   be used to bring other nodes up to date that are only slightly behind, without having to send the
   full snapshot. This is a cluster wide value and should be consistent across nodes in the cluster.
   Otherwise the value from the most recently started controller will win.
+- `warnWhenLeaderlessFor` - (optional, 1m) - Emits a warning log message if a controller is part of
+   a cluster with no leader for a duration which exceeds this threshold. 
 
 ```text
 raft:
-  bootstrapMembers:
+  initialMembers:
```

Comment:
I think we're going to replace the `raft` and `initialMembers` words with something else, and if we're representing any cluster member's advertised addresses I prefer to emphasize the importance of FQDNs, and for readability, e.g.,

```yaml
cluster:
    initialMembers:
        - tls:ctrl1.ziti.example.com:1280
        - tls:ctrl2.ziti.example.com:1280
        - tls:ctrl3.ziti.example.com:1280
```

I do not expect it will be popular to deploy cluster members on port 6262/TCP, but to instead adopt the single-port MO with 443/TCP or 1280/TCP.